### DISCUSSION TITLE
Limit user access only to studies, uploaded by him

### DISCUSSION THREAD

**Discussion by napest** [dcm4che/dcm4chee-arc-light#3756] [Q&A] [answered]
Hi! I wonder whether it is possible to implement this use case with dcm4chee:

- Admin creates a user
- User logs in to dcm4chee and his study list is empty (he sees no studies)
- He uploads some studies (ex. CT/MRI scan)
- His study list consists of studies, uploaded by him, he can view them now and interact with them
At the same time, there are lots of other users in the system, but they see only their studies

If it is possible, how should I do that?
Thank you, any help is highly appreciated!

**Comment by vrindanayak** [comment, marked as answer]
Archive supports [Access Control based on Archive AEs](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Access-control-based-on-Archive-AEs), but there isn't any access control by users. Implicitly, you could make use of the mentioned feature for a **group of users** by assigning each group an [application entity](https://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/networkAE.html)

**Comment by Oblynx** [comment]
Hi there! Thanks for the answer :smile:  We're trying to do the same, but some of the steps aren't very clear to me.

What we understand so far: [Access control based on Archive AEs](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Access-control-based-on-Archive-AEs) boils down to 2 settings:
- Access Control IDs can be assigned to an Application Entity
- Store Access Control IDs can be assigned to Studies
Then, the Application Entity will only display those Studies that have a matching Access Control ID.
Is this correct?

What we don't understand though is how to couple Application Entities with user groups (eg. on LDAP).
- Is there a way to restrict the AEs that a group of users can see?
- How does dcm4chee know the relevant group of the current user?
   - Is this through a custom OIDC claim that is mapped to an LDAP property?

This discussion should hopefully be useful to many people!

**Reply by Oblynx** [reply]
Hi @vrindanayak ! Since you answered the first question on this thread, I was wondering if you have any more valuable insight to share :smile:
